control_panel_api/auth0.py

```diff
@@ -36,34 +38,50 @@ def management_api(self):
 
         return self.api.management
 
-    def get_group_members(self, app_name):
-        group = self.authorization_api.get(Group(name=app_name))
+    def get_group_members(self, group_name):
+        group = self.authorization_api.get(Group(name=group_name))
 
         if group is None:
             return None
 
         return group.get_members()
 
-    def add_group_member(self, group_name, email):
-        group = self.authorization_api.get_or_create(Group(name=group_name))
+    def add_group_members(self, group_name, emails, user_options={}):
+        group = self.authorization_api.get(Group(name=group_name))
 
-        user = self.authorization_api.get(User(
-            email=email,
-        ))
+        all_users = self.authorization_api.get_all(User)
+        user_lookup = {
+            user['email']: user
+            for user in all_users if 'email' in user}
 
-        if user is None:
-            user = self.management_api.create(User(
-                connection='email',
-                email=email,
-                email_verified=True,
-            ))
+        def has_options(user):
+            return all(
+                item in user.items()
```

Comment:
I don't know if it's worth be more explicit and rename `has_options()` to `has_all_options()` and `item` => `option` (or maybe `user_option)`, e.g.

```python
def has_all_options(user):
    return all(
        option in user.items()
        for option in user_options.items())
```

Reply:
Not sure changing the name is necessary. Changing `item` to `option` is incorrect, because the `user.items` are not options.